lib/operations/close.js

```diff
@@ -23,13 +23,22 @@ class CloseOperation extends OperationBase {
       client.removeAllListeners('close');
       callback(err, null);
     };
+    const mongocryptdClientClose = err => {
+      const mongocryptdClient = client.s.mongocryptdClient;
+      if (!mongocryptdClient) {
+        completeClose(err);
+        return;
+      }
+
+      mongocryptdClient.close(force, err2 => completeClose(err || err2));
```

Comment:
What happens if `err` and `err2` exist?

Reply:
IMO we should prioritize the first `err`